components/builder-api/src/server/resources/origins.rs

```diff
@@ -35,28 +36,18 @@ use hab_net::{ErrCode, NetOk};
 
 use protocol::originsrv::*;
 
+use db::models::origin::{
+    CreateOrigin as CreateOriginMod, CreateOriginHandlerReq, GetOrigin as GetOriginMod,
```

Comment:
Do we need the aliases?

Reply:
unfortunately yes, there are conflicting names in the originsrv protos